README.md

```diff
@@ -26,6 +26,11 @@ Then add it to your module's `imports`.
 <json-editor [schema]="mySchema" [record]="myRecord" (onRecordChange)="doStuffWithNewRecord($event)"></json-editor>
 ```
 
+- `schema` : valid json-schema for the record.
+- `record` : valid json to be edited.
+- `onRecordChange` emitted when record change, `$event` is the edited record.
```

Comment:
It has to be called event `$event` in `html`, because it something that is provided by `angular2`. I just renamed the ones that are in `typescript`.